kebechet/managers/thoth_advise/thoth_advise.py

```diff
@@ -316,6 +316,15 @@ def run(self, labels: list, analysis_id=None):
                     runtime_environments = [
                         e["name"] for e in thoth_config["runtime_environments"]
                     ]
+                    managers = thoth_config.get("managers")
+                    disallow_run_times = managers[0].get("disallow", None)
```

Comment:
This should be per manager, not global to all managers

Reply:
ack, thanks